.github/workflows/ci.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-${{ matrix.go-version }}-go-
     - name: Set up Cloud SDK
-      uses: google-github-actions/setup-gcloud@master
+      uses: google-github-actions/setup-gcloud@v0
```

Comment:
Can you help me understand why "uses" v0 only is sufficient? I'm trying to make sure we don't lose CI coverage. In other words, why not using two workflows one for v0 (like staging?) and another one for master?

Reply:
setup-gcloud github action repo is renaming their `master` branch so they're suggesting to pin to v0 instead: https://github.com/google-github-actions/setup-gcloud#-notices

Reply:
I see. It's an official rename. LGTM.